### PR TITLE
add vtiger org and source

### PR DESCRIPTION
**Pull request by shwetagninkarad**

### CODE REVIEW

File: sources/vtiger [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "VTIGER_CRM",
+  "name": "Vtiger CRM",
+  "categories": ["CRM"],
+  "organization": "Vtiger CRM",
```
Comment (AnuFeb) [inline sources/vtiger:5]: Organization Ids must be defined in the [organizations](https://github.com/googledatastudio/ds-data-registry/tree/master/organizations) registry file.

File: sources/vtiger [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "VTIGER_CRM",
```
Comment (AnuFeb) [inline sources/vtiger:2]: The filename should match the Id property of the source definition.
Refer https://github.com/googledatastudio/ds-data-registry for details.

File: sources/vtiger (hunk 1 above)
Comment (shwetagninkarad) [reply sources/vtiger:5]: I have changed the filename to vtigercrm

File: sources/vtiger (hunk 2 above)
Comment (shwetagninkarad) [reply sources/vtiger:2]: I have changed the file name to vtigercrm.

File: sources/vtiger (hunk 1 above)
Comment (shwetagninkarad) [reply sources/vtiger:5]: I have used same organization id, that mentioned in vtigercrm organization file. Can you please check it?

File: organizations/vtigercrm [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": "VTIGER_CRM",
```
Comment (AnuFeb) [inline organizations/vtigercrm:2]: The filename should match the Id property of the source definition. Please refer this for more info https://github.com/googledatastudio/ds-data-registry

File: sources/vtigercrm [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "VTIGER_CRM",
```
Comment (AnuFeb) [inline sources/vtigercrm:2]: The filename should match the Id property of the source definition. Please refer this for more info https://github.com/googledatastudio/ds-data-registry

Comment (shwetagninkarad) [reply sources/vtigercrm]: Changed file name of source and organization to vtiger_crm as given in registry. Can you please check it ?

**Pull request closed without merging**